webpack.config.js

```diff
@@ -58,16 +58,27 @@ var configClient = {
     exprContextRegExp: /$^/,
     exprContextCritical: false,
     wrappedContextCritical: true,
-    rules: [{
-      test: /\.(jsx|js)$/,
-      use: [{
-        loader: "babel-loader"
+    rules: [
+      {
+        test: /\.ts$/,
+        enforce: "pre",
+        loader: "tslint-loader",
+        options: {
+          typeCheck: false,
+          configFile: false,
+          failOnHint: true
+        }
+      },
+      {
+        test: /\.(jsx|js)$/,
```

Comment:
We have to find a way. babel-loader is redundant with ts-loader (as javascript is a subpart of typescript). 
I will make a try if you don't find a solution.
Question: in our api, every file is written in typescript. As we exclude dependencies from webpack merge, why do we need to do something something special for js and jsx files ?